src/science/canopy/cbl_friction_vel.F90

```diff
@@ -58,18 +107,32 @@ END SUBROUTINE comp_friction_vel
 
 
 FUNCTION psim(zeta, mp, CPI_C ) RESULT(r)
+  !*## Purpose
+  !
+  ! Evaluates the integrated similarity function for momentum transfer,
+  ! \(\psi_m(\xi)\).
+  ! Uses the Businger-Dyer form for unstable conditions (\(\xi<0\)) and the
+  ! Beljaars-Holtslag form for stable conditions (\(\xi>0\)).
+  !
+  ! This function is used in the evaluation of the friction velocity in
+  ! [[comp_friction_vel]].
+  !
+  !## References
+  ! - [Beljaars and Holtslag (1991)](https://doi.org/10.1175/1520-0450(1991)030<0327:FPOLSF>2.0.CO;2)
+  ! - [Businger et al. (1971)](https://doi.org/10.1175/1520-0469(1971)028<0181:FPRITA>2.0.CO;2)
+  ! - [Dyer (1974)](https://doi.org/10.1007/BF00240838)
 
 ! mrr, 16-sep-92 (from function psi: mrr, edinburgh 1977)
 ! computes integrated stability function psim(z/l) (z/l=zeta)
 ! for momentum, using the businger-dyer form for unstable cases
 ! and the Beljaars and Holtslag (1991) form for stable cases.
 
-INTEGER, INTENT(IN) :: mp
-REAL, INTENT(IN), DIMENSION(mp) ::  zeta       !
-REAL, INTENT(IN) :: CPI_C  
+INTEGER, INTENT(IN) :: mp                !! size of cable vector of land points (-)
+REAL, INTENT(IN), DIMENSION(mp) ::  zeta !! IN current value of stability parameter \(\xi\) (-)
+REAL, INTENT(IN) :: CPI_C                !! PI (=3.14...) (-)
 
 ! function result
-REAL, DIMENSION(mp) :: r
+REAL, DIMENSION(mp) :: r                 !! OUT \(\psi_m (\xi) \)
```

Comment:
```suggestion
REAL, DIMENSION(mp) :: r                 !! \(\psi_m (\xi) \)
```
"OUT" is redundant. FORD identifies this as the result of the function. Different from subroutines.

Reply:
Actually, the documentation for the return value does not seem to work quite well in this case. It might be better to leave it outside the inline documentation (simple comment here). 

It seems FORD doesn't pick up the documentation as a description of the result since it is identified at the declaration of the function. In other word, it's better to leave the description in the first line of the purpose of the function as it is done now.